tests/smartclone_test.go

```diff
@@ -53,7 +54,29 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]SmartClone tests",
 			// Wait for operation Succeeded
 			waitForDvPhase(cdiv1.Succeeded, dataVolume, f)
 			verifyEvent(controller.CloneSucceeded, dataVolume.Namespace, f)
+			// Cleanup
+			err := utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, dataVolume.Name)
```

Comment:
Since we are improving the tests, lets take the cleanup out of the test itself, and put it in the AfterEach. Right now if the test fails above after creating the DV is not cleaned up. (It will get cleaned up when namespace is deleted by the framework, but lets not count on that working all the time)

1. Add var datavolume *cdiv1.DataVolume or put both datavolume and pvc inside a var ( ... ) block
2. In the AfterEach, check if datavolume != nil and delete it like here.
3. Set it to nil